spec/models/post_spec.rb

```diff
@@ -35,5 +35,16 @@
         expect(post.up_votes).to eq(1)
       end
     end
+
+    describe "#save_with_initial_vote" do
+      it "saves a post only if the initial vote saves as well" do
+        post = described_class.new(title: "Check One Two",
+                        body:  "Sibilance. Sibilance.",
+                        user:  authenticated_user,
+                        created_at: Time.new)
```

Comment:
@Enriikke so check out this test. Because I'm not able to use `associated_post` (it saves the post), this seems to be a decent way to initialize a post, run the `save_with_initial_vote`, and check to see if it worked. Would you approach this differently? I don't __think__ we have a need for an `associated_post_unsaved` helper but maybe there's a better way?